utils/gyb.py

```diff
@@ -464,10 +464,14 @@ def tokenGenerator(self, baseTokens):
                 if (kind == 'gybBlockOpen'):
                     # Absorb any '}% <optional-comment> \n'
                     m2 = gybBlockClose.match(self.template, closePos)
-                    assert m2, "Invalid block closure" # FIXME: need proper error handling here.
+                    if m2 is None:
+                        raise ValueError("Invalid block closure")
                     nextPos = m2.end(0)
                 else:
-                    assert kind == 'substitutionOpen'
```

Comment:
This assert is just an assert that would catch a logic error.  Please revert this part.